openpocket/alpha_spheres/alpha_spheres.py

```diff
@@ -126,27 +125,44 @@ def remove_alpha_spheres(self, indices):
         --------
 
         """
-
-        pass
-
-    def remove_small_alpha_spheres(self, minimum_radius):
-
-        """
-
-        """
-
-        indices_to_remove = np.where(self.radii < minimum_radius)
-        self.remove(indices_to_remove)
+        if indices is not None:
+        #    if isinstance(indices, (list,np.ndarray,dtype:ints)): 
+            if isinstance(indices, (list,np.ndarray)):
+                pass
+            else:
+                raise ValueError("The argument points needs to be a List, tuple or numpy.ndarray object with the integer numbers corresponding to the alpha-sphere")
+
+            self.indices = indices  # a way to store the input indices
+            alpha_sphere_index=[]
+            for ia,alpsp in enumerate(self.points_of_alpha_sphere):
+                alpha_sphere_index.append(ia) 
```

Comment:
is this the same as:
```python
alpha_sphere_index = list(range(self.n_alpha_sphere))
```
?

Reply:
Another mistake: I just try to define a new variable, in order to append the index.